server/testsupport/createInductionDtoTestDataBuilder.ts

```diff
@@ -22,9 +22,9 @@ const aCreateLongQuestionSetInductionDto = (options?: {
     workOnRelease: {
       hopingToWork: HopingToGetWorkValue.YES,
       affectAbilityToWork: [AbilityToWorkValue.NONE],
-      affectAbilityToWorkOther: null,
-      notHopingToWorkReasons: null,
-      notHopingToWorkOtherReason: null,
+      affectAbilityToWorkOther: undefined,
+      notHopingToWorkReasons: undefined,
+      notHopingToWorkOtherReason: undefined,
```

Comment:
It doesn't really matter if these are `null` or `undefined`, I'm just making them consistent.